ee-9/pom.xml

```diff
@@ -3032,6 +3032,19 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+
+            <dependency>
+                <groupId>${ee.maven.groupId}</groupId>
+                <artifactId>wildfly-webservices-opensaml-sysconfig</artifactId>
+                <version>${ee.maven.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
This should be located in boms/standard-ee/pom.xml.

When you add it please note that that pom has its dependencies strictly ordered.

Reply:
Moved to boms/standard-ee/pom.xml.